python/ray/util/spark/utils.py

```diff
@@ -332,10 +332,20 @@ def _get_avail_mem_per_ray_worker_node(
     object_store_memory_per_node,
 ):
     num_cpus = _get_cpu_cores()
+
+    if num_cpus_per_node > num_cpus:
+        raise ValueError(
+            "cpu number per Ray worker node should be <= spark worker node CPU cores."
```

Comment:
In the error message, let's print out the value of `num_cpus_per_node` and `num_cpus`